packages/common/src/Utility.ts

```diff
@@ -315,21 +322,22 @@ export function template(tpl: string, _args) {
         }
     }
 
-    return tpl.replace(nargs, function replaceArg(match, i, index) {
-        let result;
-
-        if (tpl[index - 1] === "{" &&
-            tpl[index + match.length] === "}") {
-            return i;
-        } else {
```

Comment:
@jbrundage this code was removed in your PR - I couldn't work out why?

Reply:
That condition allowed for the possibility of dynamic template inputs by calling `template` recursively until no more variables remain. Since we are now giving double curly brackets a specific meaning then that condition (which wasn't being used) is no longer viable.